pkg/cloudprovider/aws/helpers.go

```diff
@@ -51,3 +52,17 @@ func GetBucketRegion(bucket string) (string, error) {
 
 	return "", errors.New("unable to determine bucket's region")
 }
+
+// IsValidS3URLScheme returns true if the scheme is http:// or https://
+// and the url parses correctly, otherwise, return false
+func IsValidS3URLScheme(s3URL string) bool {
+	u, err := url.Parse(s3URL)
+	if err != nil {
+		return false
+	}
+	if u.Scheme == "http" || u.Scheme == "https" {
+		return true
+	} else {
```

Comment:
A couple comments minor in consequence but with the potential to improve readability:
1. The `else` here is unnecessary.
2. I also propose to switch and say `if u.Scheme != "http" && u.Scheme != "https" ` then return `false`, lastly return `true`.

Reply:
that's a good suggestion, thanks!